content/millennium/r4/foundation/other/health-cards.md

```diff
@@ -43,11 +43,11 @@ Represents the patient and the clinical data related to a vaccination
 
 Contains the following resources:
 
-* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-dm.html#tab-ms) profile with the following fields, if valued:
+* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://hl7.org/fhir/uv/shc-vaccination/2021Sep/) profile with the following fields, if valued:
```

Comment:
So, this is better that you are linking to the 2021Sep version however, the link we are replacing was pointing to a specific profile for vaccine-credential-patient-dm which would be this page in the 2021Sep version http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-patient-general-dm.html

Reply:
> So, this is better that you are linking to the 2021Sep version however, the link we are replacing was pointing to a specific profile for vaccine-credential-patient-dm which would be this page in the 2021Sep version http://hl7.org/fhir/uv/shc-vaccination/2021Sep/StructureDefinition-shc-patient-general-dm.html I think we need to link to the specific profile where warranted.

Fixed in commit https://github.com/cerner/fhir.cerner.com/pull/874/commits/6891a9e92a9faf1db82a825354e912180f12c6af